cmd/argo/commands/clustertemplate/list.go

```diff
@@ -22,6 +22,15 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list cluster workflow templates",
+		Example: `# List Cluster Workflow Templates:
+  argo cluster-template list
+	
+# List Cluster Workflow Templates in "wide" output format:
```

Comment:
This is an example, so we should be more specific and not just reiterate the option.

For example, [this other PR](https://github.com/argoproj/argo-workflows/pull/12076/files#diff-f9df5c1e05ae7460690c91a7d11839e40bf0eba4ee12c8b91e6a3d6037a4b741R85) specified what other information `wide` provides vs the default.
You should similarly test to see what it does and be more specific